[Feat] support Font family

5anniversary opened the pull request:
- Added the ability to add a font family type.

zhgchgli0718 commented:
Hello, thank you for your PR contribution.

Regarding the implementation of font-family, I have already implemented it in PR #29 (taking into account the various cases of CSS font-family and matching with UIFont/NSFont Family).

I will adjust the Code Format and CSSProperty contributions from your PR, extract them, and then merge them.

Thank you again for your contribution.

5anniversary commented:
Fixed a conflict

5anniversary commented:
@zhgchgli0718 
Is there anything else we need to do to merge this PR?

5anniversary commented:
I had some code I wanted to add, so I added it and raised it to a new PR!
I removed and uploaded it because I think it's a personal preference, but if you like it, I'll apply it in bulk to #30 PR 🙏 

And I'll close this PR!!!
